backends/bmv2/common/JsonObjects.h

```diff
@@ -49,7 +49,7 @@ class JsonObjects {
     void add_parser_transition(const unsigned id, Util::IJson* transition);
     void add_parser_op(const unsigned id, Util::IJson* op);
     void add_parser_transition_key(const unsigned id, Util::IJson* key);
-    void add_parse_vset(const cstring& name, const unsigned size);
+    void add_parse_vset(const cstring& name, const unsigned& size, const big_int& n);
```

Comment:
The change from `const unsigned` to `const unsigned&` is pointless, and I believe doesn't match the rest of the code base